Formula/monkeysphere.rb

```diff
@@ -48,6 +53,35 @@ def install
     # Consequently, this is a bit of a naughty hack but the least worst option.
     inreplace pkgshare/"keytrans", "#!/usr/bin/perl -T",
                                    "#!/usr/bin/perl -T -I#{libexec}/lib/perl5"
+
+    # find is used with GNU only permissions mode
+    inreplace pkgshare/"common", "$(find ",
+                                 "$(gfind "
+    # use liblockfile
+    inreplace pkgshare/"common", "lockfile-create",
+                                 "dotlockfile"
+    inreplace pkgshare/"common", "lockfile-touch",
+                                 "dotlockfile -T"
+    inreplace pkgshare/"common", "lockfile-remove",
+                                 "dotlockfile -u"
+
+    # Needs BASH 4.0 or later, Apple's BASH 3.2 is insufficient
+    inreplace bin/"monkeysphere", "#!/usr/bin/env bash",
+                                  "#!/usr/local/opt/bash/bin/bash"
+    inreplace sbin/"monkeysphere-host", "#!/usr/bin/env bash",
+                                        "#!/usr/local/opt/bash/bin/bash"
+    inreplace sbin/"monkeysphere-authentication", "#!/usr/bin/env bash",
+                                                  "#!/usr/local/opt/bash/bin/bash"
+  end
+
+  def caveats
+    <<~EOS
+      This formula installs BASH as a dependency because Monkeysphere requires a newer 
```

Comment:
I think this should be a caveat in the bash formula or not at all.

Reply:
I think it is worth repeating in this formula and anything else that has bash as a dependency as people may not notice the caveat streaming by on the dependency install.

Reply:
Caveats are gathered at the end of all install output, I don't think they're gonna read this one if they won't read the bash one.